src/index.js

```diff
@@ -15,12 +15,12 @@ export default class SpeechToText {
   */
   constructor(onAnythingSaid, onFinalised, onFinishedListening, language = 'en-US') {
     // Check to see if this browser supports speech recognition
-    if (!('webkitSpeechRecognition' in window)) {
-      throw new Error("This browser doesn't support speech recognition. Try Google Chrome.");
+    if (!('webkitSpeechRecognition' in window) || !('SpeechRecognition') in window) {
```

Comment:
This line should read

```
    if (
      !('webkitSpeechRecognition' in window) &&
      !('SpeechRecognition' in window)
    ) {
```

Notice the `&&`. We only want to throw an error if both properties don't exist on the windows object.

Also you want to negate `('SpeechRecognition' in window)`, not just `('SpeechRecognition')`.
`!('SpeechRecognition') in window` returns false
`!('SpeechRecognition' in window)` returns true

Reply:
I'll make the changes and update the pr.

Reply:
I have updated the changes. And I am really sorry for the delay.